### DISCUSSION TITLE
Webmin reports wrongly MySQL version

### DISCUSSION THREAD

**Discussion by samsisamsi** [webmin/webmin#2017] [General]
Webmin 2.102 reports wrongly -1 for MySQL version.
Any idea?
![Webmin](https://github.com/webmin/webmin/assets/145586392/17b19d06-8bf0-4155-8b3e-804731b88da1)

**Comment by chris001** [comment]
Looks like ClamAV and MySQL are both showing wrong info?
Same OS and version here is showing good info:
![ubuntu software versions](https://github.com/webmin/webmin/assets/259416/68e228fd-16c7-49c1-86ca-df2d7c39d050)

**Comment by iliaross** [comment]
> Webmin 2.102 reports wrongly -1 for MySQL version.
> Any idea?

Do you have MariaDB or MySQL installed? What is the output of `mysql --version` command on your system?

**Reply by samsisamsi** [reply]
mysql  Ver 8.0.34-0ubuntu0.22.04.1 for Linux on x86_64 ((Ubuntu))

**Comment by iliaross** [comment]
What is the output of `/etc/webmin/mysql/version` file? If you try to delete it and re-visit MySQL Databases Module, does it fixes the problem for you on the dashboard, after you click refresh button at the top right of the page?

**Reply by samsisamsi** [reply]
The output is 8.0.34-0ubuntu0.
Thanks, problem solved!
![Webmin](https://github.com/webmin/webmin/assets/145586392/1b9264cf-2de6-4e31-95bd-73a263b4f97f)
